src/pipeline_v3/cli/management.py

```
@@ -674,15 +674,27 @@ def _resolve_sources(
         return unique_resolved
 
     def _parse_metadata(self, metadata_list: list[str] | None) -> dict[str, Any]:
-        """Parse metadata key=value pairs."""
+        """Parse metadata key=value pairs with input sanitization."""
+        from utils.security import InputSanitizer
```

Pylint (3.3.7)

[error] Unable to import 'utils.security'

(E0401)